release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -849,6 +864,10 @@ Support for empty files using the `--registry-config` and `--to` flags in `oc re
 [id="ocp-4-10-removed-features"]
 === Removed features
 
+{product-title} 4.10 removes the Jenkins Operator, which was a Technology Preview feature, from the OperatorHub page in the {product-title} web console interface. Bug fixes and support are no longer available.
```

Comment:
@adambkaplan @akram @jkhelil WDYT about the statement that "Bug fixes and support are no longer available."